platformio.ini

```diff
@@ -9,39 +9,8 @@
 # (use `platformio_override.ini` when building for your own board; see `platformio_override.ini.sample` for an example)
 # ------------------------------------------------------------------------------
 
-# CI binaries
-; default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, esp32dev, esp32_eth # ESP32 variant builds are temporarily excluded from CI due to toolchain issues on the GitHub Actions Linux environment
-default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, esp32dev, esp32_eth, esp32dev_audioreactive, lolin_s2_mini, esp32c3dev, esp32s3dev_8MB, esp32s3dev_8MB_PSRAM_opi
-
-# Release binaries
-; default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, esp32dev, esp32_eth, lolin_s2_mini, esp32c3dev, esp32s3dev_8MB
-
-# Build everything
-; default_envs = esp32dev, esp8285_4CH_MagicHome, codm-controller-0_6-rev2, codm-controller-0_6, esp32s2_saola, d1_mini_5CH_Shojo_PCB, d1_mini, sp501e, nodemcuv2, esp32_eth, anavi_miracle_controller, esp07, esp01_1m_full, m5atom, h803wf, d1_mini_ota, heltec_wifi_kit_8, esp8285_H801, d1_mini_debug, wemos_shield_esp32, elekstube_ips
-
-# Single binaries (uncomment your board)
-; default_envs = elekstube_ips
-; default_envs = nodemcuv2
-; default_envs = esp8266_2m
-; default_envs = esp01_1m_full
-; default_envs = esp07
-; default_envs = d1_mini
-; default_envs = heltec_wifi_kit_8
-; default_envs = h803wf
-; default_envs = d1_mini_debug
-; default_envs = d1_mini_ota
-; default_envs = esp32dev
-; default_envs = esp8285_4CH_MagicHome
-; default_envs = esp8285_H801
-; default_envs = d1_mini_5CH_Shojo_PCB
-; default_envs = wemos_shield_esp32
-; default_envs = m5atom
-; default_envs = esp32_eth
-; default_envs = esp32dev_qio80
-; default_envs = esp32_eth_ota1mapp
-; default_envs = esp32s2_saola
-; default_envs = esp32c3dev
-; default_envs = lolin_s2_mini
+# CI/release binaries
+default_envs = nodemcuv2, esp8266_2m, esp01_1m_full, esp32dev, esp32_eth, esp32dev_audioreactive, lolin_s2_mini, esp32c3dev, esp32s3dev_8MB, esp32s3dev_8MB_PSRAM_opi, esp32_wrover
```

Comment:
`esp32_wrover` as default env?
Is it used that much?
We didn't have it as a binary for release 0.14.1? But also `lolin_s2_mini` was not in release 0.14.1.
Why do we keep them?

Reply:
Indeed I introduced it here as there were quite a few complaints in the past about crashing ESP32 chips (Pico in particular) since WROVER uses GPIO16 for PSRAM access.
It should become standard in the future for such builds.

ESP32-S2 and ESP32-S3 do not share PSRAM GPIO with anything else so are safe as is.

Reply:
@blazoncek Does the API have any distinction between normal ESP32 and ESP32 wrover?